### DISCUSSION TITLE
External Libraries - Access to other's libraries question

### DISCUSSION THREAD

**Discussion by melchor629** [immich-app/immich#13081] [Q&A] [answered]
Hello, the thing I am bothered is that another's external library shows up in my user's timeline (it is admin tho). I have been searching through documentation and discussions here, but I did not managed to find something useful.

Let me explain briefly:
- I created recently a new user in my instance.
- I checked that this user had no access to my own external library.
- Then added a new external library owned by the new user.
- Let Immich sync the new folder and photos appear in the new user's timeline.
- Suddenly, those photos also appeared in my timeline (?)

(edit) Note that partner sharing is not enabled between both users.

I'm starting to think that I can see all external libraries assets, because my user is admin. Is that true? Should I do something else to hide them? Or is that an issue?

Thanks :)

**Comment by bo0tzz** [comment]
Is partner sharing with this user enabled?

**Reply by melchor629** [reply]
Oh, forgot to add that. It is disabled. Neither the other user nor me set the partner sharing.

**Reply by bo0tzz** [reply]
Can you post the output of https://my.immich.app/api/partners?direction=shared-with, https://my.immich.app/api/partners?direction=shared-by and https://my.immich.app/api/libraries ?

**Reply by melchor629** [reply, marked as answer]
https://my.immich.app/api/partners?direction=shared-with
```json
[]
```

https://my.immich.app/api/partners?direction=shared-by
```json
[]
```

https://my.immich.app/api/libraries
```json
[
  {
    "id": "ee26bce0-bf85-4455-bc7b-67823e0651b1",
    "ownerId": "d04fa585-817e-42f8-84f1-1b97065376b7",
    "name": "Fotos Melchor",
    "createdAt": "2024-02-28T15:20:25.642Z",
    "updatedAt": "2024-10-01T07:02:18.996Z",
    "refreshedAt": "2024-10-01T00:00:01.920Z",
    "assetCount": 0,
    "importPaths": [
      "/mnt/media/chlm0r",
      "/mnt/media/aida"
    ],
    "exclusionPatterns": []
  },
  {
    "id": "a654328a-17c0-41b7-816a-9de620ca3f15",
    "ownerId": "3e6f9496-363e-4c6c-b71b-5658e30bb3a9",
    "name": "Fotos Aida",
    "createdAt": "2024-09-30T17:41:20.920Z",
    "updatedAt": "2024-10-01T00:00:01.974Z",
    "refreshedAt": "2024-10-01T00:00:01.972Z",
    "assetCount": 0,
    "importPaths": [
      "/mnt/media/aida"
    ],
    "exclusionPatterns": [
      "**/@eaDir/**",
      "**/._*"
    ]
  }
]
```

Looking into the third response, I can see now the issue 😅 Removed the path already and it is fine now.

That might explain why when I was configuring the external library, the path was not added to the new external library. So, for some reason, it was added to the first external library (either by my mistake or by another thing). Either way, that would explain the issue at all.

Thanks for the the assistance :)
